app/controllers/schedule_controller.rb

```diff
@@ -2,33 +2,38 @@
 
 class ScheduleController < ApplicationController
   def save_scheduled_publishing_datetime
-    Edition.find_and_lock_current(document: params[:document]) do |edition|
-      checker = Requirements::ScheduledDatetimeChecker.new(permitted_params)
-      issues = checker.pre_submit_issues
-
-      if issues.any?
-        flash["alert_with_items"] = {
-          title: I18n.t!("requirements.scheduled_datetime.title"),
-          items: issues.items(
-            link_options: {
-              scheduled_datetime: { href: "#scheduled_publishing_datetime" },
-            },
-          ),
-        }
-        flash[:scheduled_publishing_params] = permitted_params
-        redirect_to document_path(edition.document)
-        next
-      end
-
-      set_scheduled_publishing_datetime(edition, checker.parsed_datetime)
-
+    result = Schedule::SaveDatetimeInteractor.call(params: params, user: current_user)
+    edition, issues = result.to_h.values_at(:edition, :issues)
+
+    if issues
+      flash.now["alert_with_items"] = {
+        "title" => I18n.t!("requirements.scheduled_datetime.title"),
```

Comment:
I think this is something we might want to change since it seems tailored to a particular error.

Reply:
This is now using the generic "You must" title and the individual error messages have been adapted to work with it.

Reply:
I thought we were going with "Thou shalt" ;-).

Reply:
Well you know, that would be my preference 🤷‍♀